src/views/FilterEnergyView.vue

```diff
@@ -93,7 +94,7 @@ import { Paragraph } from "@utrecht/component-library-vue";
                                 >Zoek op trefwoord/code</label
                               >
                             </div>
-                            <input
+                            <Textbox
```

Comment:
As you're changing the CSS component to a VueJS component we don't need to specify those CSS classNames anymore.

When changing the HTML inputs to VueJS `Textbox` please also remove the `utrecht-textbox` classes